src/AzureIoTHub.Portal/Client/Pages/Devices/DeviceListPage.razor

```diff
@@ -179,6 +179,8 @@
 
     private int[] pageSizeOptions = new int[] { 2, 5, 10 };
 
+    private readonly MudTheme mudTheme = new();
```

Comment:
Might be better to get the default theme from the application instead of creating the instance here ?